packages/synapse-react-client/src/lib/utils/functions/sqlFunctions.ts

```diff
@@ -35,7 +35,8 @@ export const generateQueryFilterFromSearchParams = (
   if (!searchParams) {
     return
   }
-  const isQueryWrapperKey = (key: string) => key.startsWith('QueryWrapper')
+  const isQueryWrapperKey = (key: string) =>
+    key.startsWith('QueryWrapper') || key.endsWith('NotAQueryFilterKey')
```

Comment:
Should the suffix case be negated? Or is QueryWrapper key != QueryFilter key (in which case maybe this needs a new name?)
```suggestion
  const isQueryWrapperKey = (key: string) =>
    key.startsWith('QueryWrapper') || !key.endsWith('NotAQueryFilterKey')
```

Reply:
QueryWrapper key != QueryFilter key.  I think this is when the Query in a DetailsPage is changed (filtering takes place), which automatically updates the URL with the QueryWrapperX parameter.  But these should be ignored here, when the DetailsPage interprets every search param key/value pair as a query condition